lib/appsignal/error_handler.ex

```diff
@@ -140,6 +140,16 @@ defmodule Appsignal.ErrorHandler do
   end
 
   if Appsignal.phoenix? do
+    def extract_reason_and_message(%Phoenix.ActionClauseError{}, prefix) do
+      message = """
+      could not find a matching clause to process request.
+      This typically happens when there is a parameter mismatch but may
+      also happen when any of the other action arguments do not match.
```

Comment:
This is the latest version of the message in Phoenix? Or is the difference intentional? It's missing the controller name now.

I'm seeing a slightly different message in Phoenix 1.3.2:

```
could not find a matching MyApp.PageController.index clause
to process request. This typically happens when there is a
parameter mismatch but may also happen when any of the other
action arguments do not match. The request parameters are:

  %{foo: "filtered!"}
```

Reply:
We can't get the controller name from the exception struct, because it only includes a message. Do you get that message *with* this patch?